### PR TITLE
Fixes #5272

### LINKED ISSUE #5272: (WEB REPORT BY: blarghargin123 REMOTE: 198.245.63.50:7777) Gyrotron cant be rotated
>Revision 436e36750b7d5e10d03f1fe970ec02a43624609c

>General description of the issue
Gyrotron cant be rotated unless its pulled into the direction you want

>What you expected to happen
I would be able to rotate the damn thing

>What actually happened
I couldnt rotate it

>Steps to reproduce if possible
Fix yo shit Cael Aislinn. Oh wait, your dead.

### PR DESCRIPTION
**Pull request by PJB3005**

### CODE REVIEW

File: code/WorkInProgress/Cael_Aislinn/Rust/gyrotron.dm [hunk 1]
```diff
@@ -90,3 +90,31 @@
...
+	set category = "Object"
+
+	if(usr.restrained() || usr.stat || usr.weakened || usr.stunned || usr.paralysis || usr.resting || !Adjacent(usr))
+		return
+
+	if(anchored)
+		usr << "<span class='notify'>\the [src] is anchored to the floor!</span>"
+		return
+
+	dir = turn(dir, -90)
```
Comment (clusterfack) [inline code/WorkInProgress/Cael_Aislinn/Rust/gyrotron.dm]: Wont a -90 degree turn make it go counter clockwise? Are we looking at this from a compass rose perspective or a grid?

Comment (PJB3005) [inline code/WorkInProgress/Cael_Aislinn/Rust/gyrotron.dm]: If you look at emitter code, [it's also -90 for CW](https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/modules/power/singularity/emitter.dm#L51)

Comment (ComicIronic) [inline code/WorkInProgress/Cael_Aislinn/Rust/gyrotron.dm]: Then it would be wrong.
To quote the DM ref:
 `dir = turn(NORTH, 90) // dir = west`

Comment (PJB3005) [inline code/WorkInProgress/Cael_Aislinn/Rust/gyrotron.dm]: er, going NORTH -> WEST is CCW, and as you can see CCW uses 90, CW uses -90

**Pull request merged**